ci/create-cuvs-multiarch-manifest.sh

```diff
@@ -1,24 +1,26 @@
 #!/bin/bash
 
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
 set -eEuo pipefail
 
 common_path="$(dirname "$(realpath "$0")")/common.sh"
 # shellcheck source=common.sh
 source "$common_path"
 
 cuvs_bench_source_tags=()
-cuvs_bench_datasets_source_tags=()
+# cuvs_bench_datasets_source_tags=()
```

Comment:
This and other things I'm commenting out in this file are unused as long as this other code is commented out:

https://github.com/rapidsai/docker/blob/5b407711e0ab1a154ece622e530d5eab20afe77c/ci/create-cuvs-multiarch-manifest.sh#L27-L28

as it was in #725

Reply:
The PR mentioned in #725 as a patch was closed, but there is a comment that indicates this might have been fixed in 25.02, though it's unclear, see https://github.com/rapidsai/docker/pull/723#pullrequestreview-2532605785 

Is there an issue or PR in progress tracking this? that we can reference so when it's fixed we can uncomment this back?

Reply:
Thanks, good point. I just pushed https://github.com/rapidsai/docker/pull/738/commits/ca8832f8f7e10f9a703534b3aabe46db2b0685af, which adds a comment to the issue tracking the cuVS dataset stuff: https://github.com/rapidsai/docker/issues/724